SystemManager/Inc/SM.hpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * SM.hpp
+ *
+ * System Manager Header
+ *
+ * Created on: Nov 7, 2022
+ * Author(s): Gordon Fountain
+ */
+
+#ifndef ZPSW3_SM_HPP
+#define ZPSW3_SM_HPP
+
+#include <stdint.h>
+#include "cmsis_os.h"
+// #include "LOS_Link.hpp"
+// #include "LOS_Actuators.hpp"
+#include "SM_StateManager.hpp"
+
+class SystemState;
+
+namespace SM {
+    // Enumerates the current drone status
+    enum _System_Manager_Drone_Operation_Mode{BOOT, DISARMED, GROUND_OPS, FLIGHT, FATAL_FAILURE = -1};
+
+    enum _System_Manager_Cycle_Status {COMPLETED_CYCLE = 0, IN_CYCLE, FAILURE_MODE};
+}
+
+class SystemManager {
+public:
+    SystemManager();
+    void execute();
+    void setState(SystemState& newState);
```

Comment:
nit: style / naming convention.

https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/1827635203/Style+Guide

Reply:
Don't waste your time nitpicking this! This is a long way from actual review!

Reply:
Also good to note *what* the nitpick is. Eg. "tab your public:/private:"